static/src/javascripts/projects/common/modules/experiments/tests/across-the-country.js

```diff
@@ -0,0 +1,53 @@
+define([
+    'fastdom',
+    'common/utils/$',
+    'common/utils/config'
+], function (
+    fastdom,
+    $,
+    config
+) {
+
+    return function () {
+        this.id = 'AcrossTheCountry';
+        this.start = '2015-03-19';
+        this.expiry = '2015-04-19';
+        this.author = 'Grant Klopper';
+        this.description = 'Checking effect of moving "across the country" container higher up the US front';
+        this.audience = 1.0;
```

Comment:
Unless I have misunderstood how this works (quite possible) then 50% of people on the US front will see the variant and 50% will see the control. This is what I want.